AutolayoutPractice/RemindersList/Model/Reminder.swift

```diff
@@ -12,4 +12,5 @@ struct Reminder: Codable {
     var description: String
     var dueDate: Date
     var completed: Bool
+    var isPriority: Bool
```

Comment:
What does priority entail here?

Reply:
I recommend adding a description of the product requirements in the PR description